devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,71 @@
+
+However, in the same way modern Devops discourages to depend directly on any central repository, it is not advised to depend directly on ConanCenter for production environments, for several reasons:
+
+- ConanCenter accepts contributions that require the most recently released versions of Conan. If you are in an environment where you are unable to keep Conan up-to-date across all your systems (developers, CI, etc) - depending directly on recipes from ConanCenter carries the risk of retrieving recipes that do not work with your older version of Conan.
+- The changes in recipes might also unpexectedly cause bugs and other issues. Recipes and package updates might bump dependencies versions to use the latest, forcing a continuous update over all dependencies versions. That can cause version conflicts if your project doesn't bump dependencies versions at the same pace.
+- Reviews in ConanCenter can take a while to be reviewed and merged, as there is usually a high volume of requests. Please consider this if you may find yourself in a situation where you may need proposed changes merged with urgency. Pull Requests are considered according to priorities and resources, and will only be accepted with positive peer reviews by the Conan team.
+- Binary configurations can be discontinued in ConanCenter, when new compiler versions are released and ConanCenter starts to build them, it can drop some of the older versions.
+- Outages, maintenance windows and other service interruptions can happen, ranging from a few minutes to days. The infrastructure to host central repositories is typically large and complex, and relies on cloud providers that sometimes have their own issues, networks, CDN, etc.
+- Supply chain issues. Even if the best effort is made to make ConanCenter secure and the process to add packages to ConanCenter is one of the most strict ones with human reviews for every package and many automated checks and processes, eventually all systems and central repositories in every technology are subject to attempts of supply chain attacks.
+
+Most of these reasons are not unique to Conan, but to every other package manager for other programming languages. 
+
+.. important::
+  
+  The general devops known good practice is that you shouldn't rely on central repositories on production systems, but do consider the following approaches:
+
+  - If your project is mostly a "consuming" project, i.e. you are not generating your own packages, but just consuming packages from ConanCenter, and the above issues are not a concern because your project does not need that kind of robustness or security, then you can use **lockfiles** to at least guarantee that your project build using the same dependencies over time. Check :ref:`the lockfile docs<tutorial_versioning_lockfiles>` for more information.
```

Comment:
Picking the word that match the CLI so there's no ambiguity 

```suggestion
  - If your project is mostly a "consuming" project, i.e. you are not creating or uploading your own packages, but just installing packages from ConanCenter, and the above issues are not a concern because your project does not need that kind of robustness or security, then you can use **lockfiles** to at least guarantee that your project build using the same dependencies over time. Check :ref:`the lockfile docs<tutorial_versioning_lockfiles>` for more information.
```